employee_seniority_level/views/hr_employee_seniority_level.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+  <record id="hr_employee_seniority_level" model="ir.ui.view">
+    <field name="name">hr.employee.form.seniority.level</field>
+    <field name="model">hr.employee</field>
```

Comment:
Would look better to split these views and have a file for hr.employee, another one for product, and this one for seniority level.

Reply:
Personally I think, there is different way to look at this. As the file is small having all the view change in one file makes for easier reading/understanding and all changes are concerning the seniority level.
When the file becomes to large it is a good idea to split in multiple files.
I checked the guide lines :
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#xml-files
But did not see any info on this